lib/UI/UI.rb

```diff
@@ -54,9 +63,12 @@ def announce message
   end
 
   def announce_results(board, players)
-    announce("It's a tie") if board.tie?
+    if board.win?(players)
     players.each do |player|
-      announce(player.symbol + " has won!") if board.is_a_winner?(player)
+      announce(player.symbol + " has won") if board.is_a_winner?(player)
```

Comment:
I feel like this would make more sense to find the player that's won instead of iterating as if there'd be multiple players that won. Like using a loop like this makes it appear like there could be multiples.